### ISSUE TITLE
Doesn't block unless you disable/enable blocking.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#160]
```
In some unknown circumstances, PeerBlock starts up in a state such that it
doesn't actually appear to be doing anything unless you disable then
re-enable blocking.  No "allowed" packets displayed on the main window,
"ping 3.0.0.0" test fails, etc.

This is a known issue that people have been hit with for awhile now,
apparently nobody (myself included) ever got around to adding a bug to
track it.

Working on looking into this now...
```

Original issue reported on code.google.com by `peerbloc...@gmail.com` on 12 Oct 2009 at 1:20

**Comment by GoogleCodeExporter** [comment]
```
Can those of you who've hit this bug try to reproduce it with the new Beta 
Release,
r223?  I'm wondering if the fix for Issue #22 resolved this one as well...  That
would make me happy...!
```

Original comment by `peerbloc...@gmail.com` on 24 Nov 2009 at 2:39

**Comment by GoogleCodeExporter** [comment]
```
I haven't heard any recent reports of this issue, and nobody's updated it with 
any
test results from r223, so I'm going to assume it was resolved with the fix for 
Issue
#22.  If that proves to NOT be the case, we can re-open it.
```

Original comment by `peerbloc...@gmail.com` on 1 Dec 2009 at 3:27
- Changed state: **Fixed**
